mitosheet/src/mito/components/footer/SheetTabContextMenu.tsx

```diff
@@ -132,7 +112,6 @@ export default function SheetTabContextMenu(props: {
                 // doesn't compete updating the uiState to this sheet instead of
                 // the new graphID that we're creating
                 e?.stopPropagation()
-                void graphData()
```

Comment:
Hmm I think I did this but I'm confused about why this still works. I checked and it looks like it still works?